viewer/packages/360-images/src/annotation/ImageAnnotationObject.ts

```diff
@@ -49,13 +49,22 @@ export class ImageAnnotationObject implements Image360Annotation {
   private static createObjectData(detection: AnnotationData): ImageAnnotationObjectData | undefined {
     if (isAnnotationsObjectDetection(detection)) {
       if (detection.polygon !== undefined) {
-        return new PolygonAnnotationData(detection);
+        return new PolygonAnnotationData(detection.polygon);
       } else if (detection.boundingBox !== undefined) {
         return new BoxAnnotationData(detection.boundingBox);
       } else {
         return undefined;
       }
     } else if (isAnnotationAssetLink(detection)) {
+      const objectRegion = (detection as any).objectRegion;
```

Comment:
Add a TODO here and explain why we need any (so it can be fixed when SDK is ready)